docs/setup-ci.md

```diff
@@ -0,0 +1,102 @@
+
+For Azure Secure Files, retrieving the secret will download the .pfx itself. The remaining two methods will give you the encoded string. Thus, you'll have to decode the secret, and save it to a .pfx file before it can be used to sign.  See the Xaml-Islands-Samples repository or the react-native-windows repository for the powershell commands to decode and generate .pfx files within a pipeline.
+	
+Make sure to delete your .pfx from the pipeline once you've finished using it.
+
+## FAQ's
+### I'm ready to do a signed app build. How do I tell msbuild to use my certificate?
+You can tell msbuild what .pfx you want to use to sign your app using the `PackageCertificateKeyFile` msbuild property. This property expects the file path to the .pfx you want to use. See [here](https://github.com/microsoft/react-native-windows/blob/353321ee40391f6f302e7cc80f96285e12780cbe/.ado/jobs/playground.yml#L114) for an example of this using VSBuild. See [here](https://github.com/microsoft/react-native-windows/blob/353321ee40391f6f302e7cc80f96285e12780cbe/.ado/templates/run-windows-with-certificates.yml#L48) for an example of this using the RNW CLI.
+
+### I have a pipeline that runs on forks of my repository (i.e. when a PR is being made). Can I access my certificate data from this pipeline?
+No. When data is securely stored through Github Secrets, Azure DevOps Secure Files, or Azure Key Vault, it can only be run from pipelines that are executing on branches of the original repository. The data cannot be accessed from pipelines running code from repository forks. Why? Because if the data was able to be accessed, someone could manipulate the pipeline source code within their fork to retrieve the data, leaving it unsecured.
+
+### I want to build an installable copy of my app from a pipeline that doesn't have access to my certificate data. What can I do?
+This case may apply to you if you want to do some E2E testing within a PR pipeline to make sure incoming changes don't break main. You do have a couple of options here to make do without a certificate.
+
+If you are simply trying to build - but not deploy - your app, you can successfully build non-signed RNW apps by setting the msbuild argument `AppxPackageSigningEnabled` to false. See [here](https://github.com/microsoft/react-native-windows/blob/353321ee40391f6f302e7cc80f96285e12780cbe/.ado/jobs/playground.yml#L95) for an example of this using VSBuild. See [here](https://github.com/microsoft/react-native-windows/blob/353321ee40391f6f302e7cc80f96285e12780cbe/.ado/templates/run-windows-with-certificates.yml#L34) for an example of this using the RNW CLI.
+
+If you are trying to build and deploy your app for E2E testing, you can successfully do this with a non-signed RNW app by using the `--deploy-from-layout` option from the RNW CLI along with setting the msbuild argument `AppxPackageSigningEnabled` to false.
```

Comment:
If you are trying to build and deploy your app for E2E testing, you can successfully do this with a non-signed RNW app by using the `--deploy-from-layout` option from the RNW CLI along with setting the MSBuild argument `AppxPackageSigningEnabled` to false.